MREUnityRuntime/MREUnityRuntimeLib/App/MixedRealityExtensionApp.cs

```diff
@@ -474,27 +481,36 @@ void PerformUserJoin()
 				if (user == null)
 				{
 					user = userGO.AddComponent<User>();
+
 					// Generate the obfuscated user ID based on user tracking permission.
-					user.Initialize(hostAppUser, GenerateObfuscatedUserId(hostAppUser), this);
+					Guid instancedUserId = GenerateObfuscatedUserId(hostAppUser, EphemeralAppId);
```

Comment:
So is the plan to push the randomization of a user who has denied user tracking to the host app again?